main.go

```diff
@@ -105,6 +107,10 @@ func main() {
 			log.Info("Found ENV MANAGED_CONFIG, initializing using", "tool.Options.ManagedConfigFilePathName",
 				tool.Options.ManagedConfigFilePathName)
 			managedCfg, err = clientcmd.BuildConfigFromFlags("", tool.Options.ManagedConfigFilePathName)
+			if err != nil {
```

Comment:
Ideally this should have been a separate commit since it's a separate logical change that we want to keep even if we decided in the unlikely event to revert the commit introducing the health and readiness checks.

I won't block on it though.  :slightly_smiling_face: